pgtest.go

```diff
@@ -39,21 +39,24 @@ type PG struct {
 // crashes, but we don't care about that anyway during unit testing.
 //
 // Use the DB field to access the database connection
-func Start() (*PG, error) {
-	return start(false, "")
+func Start(config ...*PGConfig) (*PG, error) {
+
+	if len(config) < 1 {
+		return start(NewConfig())
+	}
+
+	return start(config[0])
 }
 
-// Starts a new PostgreSQL database
+// start Starts a new PostgreSQL database
 //
 // Will listen on a unix socket and initialize the database in the given
-// folder, if needed. Data isn't removed when calling Stop(), so this database
+// folder (cionfig.Dir), if needed.
+// Data isn't removed when calling Stop() if config.Persistent == true,
+// so this database
 // can be used multiple times. Allows using PostgreSQL as an embedded databases
 // (such as SQLite). Not for production usage!
-func StartPersistent(folder string) (*PG, error) {
-	return start(true, folder)
-}
```

Comment:
We can't remove this one as I don't want to break any existing code.

Reply:
Good catch and well noted on not changing tests. Will revert